Sample/ping.json

```diff
@@ -17,7 +17,7 @@
                     }
                 ],
                 "messagePayload": {
-                    "title": "[ToolTip] tooltip 1",
+                    "title": "[Tooltip] tooltip 1",
```

Comment:
@imaginaris did you change all occurrences of "ToolTip". Maybe we should have a mix of both forms in the unit tests.

Reply:
I added a unit test for that (https://github.com/rakutentech/ios-inappmessaging/pull/210/files#diff-e2dbcd8583148f84ae18f400d37b5cbcaffaa5b013f33bf654d4a30c237728f7R137). Do you think it's worth having more than one form in json stubs as well?

Reply:
No - what you have now is sufficient. Thanks